### ISSUE TITLE
IDMS program specific copybooks not navigable

### ISSUE THREAD

**Issue by VitGottwald** [eclipse/che-che4z-lsp-for-cobol#1518]
## Description ##
When I try to navigate to an IDMS copybook that resides in a program specific folder 

```json
"cobol-lsp.cpy-manager.idms.paths-local": [
		"copy/IDMS/${fileBasenameNoExtension}"
 ]
```

the copybook name in `COPY IDMS <copybook-name>` is not clickable.

## Expected behavior ##
I can ctrl+click on the copybook name and VSCode will navigate to it.

## Additional information ##
Replacing `${fileBasenameNoExtension}` with the program name fixes the issue.

**Comment by VitGottwald** [comment]
Interestingly enough this works with DaCo copybooks.

**Comment by VitGottwald** [comment]
Fixed by #1519.
